discord/threads.py

```diff
@@ -223,6 +223,14 @@ def mention(self) -> str:
         """:class:`str`: The string that allows you to mention the thread."""
         return f"<#{self.id}>"
 
+    @property
+    def jump_url(self) -> str:
+        """:class:`str`: The string that allows you to mention the thread.
+
+        .. versionadded:: 2.0
+        """
+        return f'https://discord.com/channels/{self.guild.id}/{self.id}'
```

Comment:
```suggestion
        return f"https://discord.com/channels/{self.guild.id}/{self.id}"
```

Fix for project linting format